### ISSUE TITLE
ProgressIndicator is silent when navigating to the progress bar

### ISSUE THREAD

**Issue by yupasz** [microsoft/fluentui#14462]
### Environment Information

- **Package version(s)**: Fluent UI React 7.126.3
- **Browser and OS versions**: 
Environment : Edge + Narrator.​
OS: Window 10 (Not reproducible in Window Server 2016)
OS version: 2004
Build version: 19640.1
Microsoft Edge (Anaheim) Version 85.0.531.1 (Official build) dev (64-bit)

### Describe the issue:

When navigating to the progress bar, the focus will first go to the whole bar but Narrator is silent. Press down arrow again, the focus will go to the blue bar with current progress value and Narrator will read the values.
If Narrator focus is moving to the silent element then users will be confusing, and it will be difficult for users with disabilities to access the UI or understand and interact with it.

### Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/progressindicator
#### Actual behavior:

In scan mode, the focus goes on the silent element after "Example title" and narrator remains silent.
![ProgressIndicator](https://user-images.githubusercontent.com/69033452/89874537-8ba88d00-dbee-11ea-8584-5c04e7c27022.gif)

#### Expected behavior:

The focus shouldn't land on the silent element after any content.

**Comment by khmakoto** [comment]
I've checked our source code and it seems that we are doing everything correctly on our end and this is the behavior of opting into scan mode itself. This happens because the inherent function of scan mode is to break out of author's  intent when it comes to navigation and dictation. Given all of this, I'll be marking this issue as `Resolution: External`. Thanks for filing it and let us know if there is anything else we can do to help!

**Comment by yupasz** [comment]
@khmakoto Thanks for your confirmation! Do you think this bug should be reported to Narrator team?

**Comment by khmakoto** [comment]
@yupasz maybe? Scan mode is really interesting because some things will get read unintentionally but that makes sense given scan mode's nature.
